python/py-kombine/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-kombine
+version             0.8.3
+
+categories-append   science
+maintainers         {uoregon.edu:bfarr @bfarr} openmaintainer
```

Comment:
@duncanmmacleod Should you also be on the maintainers list? Or is @bfarr maintaining this alone?

Reply:
I have added myself.